### PR TITLE
remaining Studio changes

### PR DESCRIPTION
**Pull request by Axelcureno**
Remaining changes needed for running RTE in merch at scale studio

Test URLs:
- Before: https://main--mas--adobecom.hlx.live/
- After: https://studio--mas--adobecom.hlx.live/

### CODE REVIEW

**Comment by yesil** [comment]
@Axelcureno can we clean bundles from libs that have been moved to Milo?

**Comment by Axelcureno** [comment]
> @Axelcureno can we clean bundles from libs that have been moved to Milo?

@yesil yes of course, I have cleaned up those files.

File: mas/src/mas.js [hunk 1]
```diff
@@ -10,14 +8,40 @@ const features = searchParams.get('features');
...
 const config = () => ({
     env: { name: envName },
     commerce: { 'commerce.env': commerceEnv },
     locale: { prefix: locale },
 });
 
-init(config);
+(async () => {
+    try {
+        await loadScript(
```
Comment (rahulgupta999) [inline mas/src/mas.js:30]: can this URL be put in a config file and based on we are running on local or hosted, we can pick the intended URL

Comment (yesil) [reply mas/src/mas.js:30]: actually it should either be www.stage.adobe.com or www.adobe.com.
We can change it later as well once we decide how to handle environments on MAS platform.

File: mas/src/mas.js [hunk 2]
```diff
@@ -10,14 +8,40 @@ const features = searchParams.get('features');
 const envName = isStage ? 'stage' : 'prod';
 const commerceEnv = isStage ? 'STAGE' : 'PROD';
 
+function loadScript(url) {
```
Comment (yesil) [inline mas/src/mas.js:11]: this should not be needed. 
We should import the file not load it this way.

**Review by yesil — CHANGES_REQUESTED** [review]

File: mas/src/mas.js (hunk 1 above)
Comment (Axelcureno) [reply mas/src/mas.js]: fixed

File: mas/src/mas.js (hunk 2 above)
Comment (Axelcureno) [reply mas/src/mas.js]: removed

**Comment by Axelcureno** [comment]
Closing this PR since https://github.com/adobecom/mas/pull/56 contains this and more changes.

**Pull request closed without merging**